examples/workflows/upstream/title-and-descriptin-with-markdown.upstream.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Nit - file name descriptin vs description

Reply:
Gonna regenerate all the examples in #982

Reply:
[Fixed in](https://github.com/argoproj-labs/hera/pull/982/commits/be703b643dac17df798e1be61a2986a9d3110e93)